.circleci/helm.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+set -eEu -o functrace
+
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+ROOT_PROJECT_DIR=$(dirname "${SCRIPT_DIR}")
+cd $ROOT_PROJECT_DIR
+SUB_PROJECTS_DIRS=$(find . -iname "helm" | sed 's/\(.*\)\/.*/\1/')
+
+script=$0
+subcommand=$1; shift
+
+case "$subcommand" in
+  validate)
+    for SUB_PROJ_DIR in $SUB_PROJECTS_DIRS; do
+      cd $SUB_PROJ_DIR
+      echo "*******"
+      echo "validating charts for:$(pwd)"
```

Comment:
```suggestion
      echo "Validating charts for dir \"$(pwd)\""
```

Wrapping the value with quotes is really helpful when having weird things like empty string.